src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -3305,6 +3310,26 @@ public function is_development_environment() {
 		return in_array( wp_get_environment_type(), array( 'development', 'local' ), true );
 	}
 
+	/**
+	 * Returns the URL used for checking page caching. By default, this is the home URL.
+	 *
+	 * @since 6.8.0
+	 *
+	 * @return string The URL for checking page caching.
+	 */
+	public function get_persistent_page_cache_url() {
```

Comment:
This shouldn't be a public method that we have to maintain. If anything it should be private, but I don't think we need an extra method for this in the first place.